spk/python3/Makefile

```diff
@@ -1,7 +1,7 @@
 SPK_NAME = python3
 SPK_SHORT_VERS = 3.5
-SPK_VERS = $(SPK_SHORT_VERS).2
-SPK_REV = 6
+SPK_VERS = $(SPK_SHORT_VERS).5
+SPK_REV = 1
```

Comment:
SPK_REV needs to increment, independent of major, minor or patch versions. Please change to 7.

Reply:
updated per both your suggestions